docs/src/running-validator/validator-reqs.md

```diff
@@ -32,6 +32,32 @@ Here are our recommendations for low, medium, and high end machine specification
 | Accounts Drive\(s\) | None                                                  | Samsung 970 Pro 1TB    | 2x Samsung 970 Pro 1TB |                                                                                        |
 | GPU                 | Nvidia 1660ti                                         | Nvidia 2080 Ti         | 2x Nvidia 2080 Ti      | Any number of cuda-capable GPUs are supported on Linux platforms.                      |
 
+## Virtual machines on Cloud Platforms
+
+You can run validator on any VMs hosted on cloud computing platforms.
+However, this is not recommended for long-term cost-efficient perspective.
+
+However, it may be convenient to run RPC nodes on VM instances for your internal
+usage. This use case includes exchanges and services built on Solana.
```

Comment:
```suggestion
While you can run a validator on a cloud computing platform, it may not
be cost-efficient over the long term.

However, it may be convenient to run non-voting api nodes on VM instances for your own internal
usage. This use case includes exchanges and services built on Solana.
```